README.md

```diff
@@ -1,109 +1,103 @@
-<span align="center"> 
+<span align="center">
+ # Checkpoint
  
-  # Checkpoint
-
 </span>
 <br />
-
+ 
 <span align="center">
-
-<div>
  
+<div>
 [![checkpoint logo](https://github.com/antrikshmisri/checkpoint/blob/master/docs/_static/logo.png?raw=true)](http://checkpoint.antriksh.live/)
```

Comment:
Can you check if this line is fine? The logo is not visible in the middle. I can only see the URL instead
![image](https://user-images.githubusercontent.com/43227329/157639185-7a09ea93-833b-4b9d-9c05-94c3b865ab33.png)